packages/story-editor/src/components/form/color/color.js

```diff
@@ -159,6 +153,7 @@ const Color = forwardRef(function Color(
           }
         >
           <EyeDropperButton
+            id={uuidv4()}
```

Comment:
just flagging again. I think you might have mentioned this having a new id on every render doing something but I can't remember?